### DISCUSSION TITLE
3 of the new icons are breaking the rule for icons

### DISCUSSION THREAD

**Discussion by rchiileea** [elax46/custom-brand-icons#753] [General]
@elax46  I have to point this out now as I dont want to see are old issues re crop up again with oversized icons, with icons we are stuck with the same rules as material design icons too due to the nature of the beast.

The box is 24x24 but the bleed is 2 pixels, so the rule should be a no go area for 2 pixels, 3 icons have broke that rule, not submitted by me.

Add ikea tokabo by @lolcatnip in https://github.com/elax46/custom-brand-icons/pull/732
Added Colombia and Stremio by @Gtt1229 in https://github.com/elax46/custom-brand-icons/pull/740

These icons break the rule and will only cause issues down the line, So I have created these to use

@Gtt1229 we appreciate your contributions, but please keep within the rule for icons here is a template

![cbi-template](https://github.com/user-attachments/assets/006368f9-ac7b-4c60-bd94-5f0f9cb3f24f)
[cbi-template.zip](https://github.com/user-attachments/files/16862557/cbi-template.zip)

both svg and ai file templates are in here to make it easier.

**Comment by elax46** [comment]
Thanks for your important contribution in the next pr I will let you evaluate this issue, in case in the next update if you can fix the added icons and insert the reference of this description from inside the redme @rchiileea

**Comment by rchiileea** [comment]
@elax46 I have already updated the two I could find, could not find the other version of stremio in the relase only the one I submited,

here is the ikea and columbia fcf redone 
[updated.zip](https://github.com/user-attachments/files/16862769/updated.zip)

**Reply by elax46** [reply]
> @elax46 I have already updated the two I could find, could not find the other version of stremio in the relase only the one I submited,
> 
> here is the ikea and columbia fcf redone [updated.zip](https://github.com/user-attachments/files/16862769/updated.zip)

maybe I think it's time to update the readme by inserting these specifications

**Reply by lolcatnip** [reply]
Please provide clear instructions for future contributors

**Reply by Gtt1229** [reply]
@rchiileea Thank you for bringing this to my attention. I will keep this in mind going forward.

**Reply by Gtt1229** [reply]
@rchiileea @elax46 

Could take a look and provide feedback for this draft pull request adding this information to the contributing page? https://github.com/elax46/custom-brand-icons/pull/759
